nilearn/glm/first_level/first_level.py

```diff
@@ -235,6 +235,32 @@ def run_glm(Y, X, noise_model='ar1', bins=100,
     return labels, results
 
 
+def _check_trial_type(events):
+    """Check that the event files contain a "trial_type" column.
+
+    Parameters
+    ----------
+    events : :obj:`list` of path of events.tsv files.
```

Comment:
this seems to be mixing type definition and description. Also clarify if the paths are strings or path-like objects (or both) in the type definition and then the description can be: "A list of paths of events.tsv files"

Reply:
ah yes.
let me fix that.